fcrepo-kernel-api/pom.xml

```diff
@@ -69,6 +69,22 @@
       <groupId>javax.servlet</groupId>
       <artifactId>javax.servlet-api</artifactId>
     </dependency>
+    <dependency>
```

Comment:
Whuh? Why is this JSON impl stuff coming into the kernal _API_? I don't like this at all.